### PR TITLE
tiramisu: 1.0 -> unstable-2021-05-20

### PR DESCRIPTION
**Pull request by moni-dz**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123703` at b3322e6f run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>tests.trivial</li>
    <li>tiramisu</li>
  </ul>
</details>

**Comment by wishfort36** [comment]
Result of `nixpkgs-review pr 123703` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>tests.trivial</li>
    <li>tiramisu</li>
  </ul>
</details>

**Review by wishfort36 — COMMENTED** [review]
The diff itself LGTM. Now that two more commits have been added, you might want to consider adding those as well. I built the package manually as well, which compiled. During testing, I managed to find a really weird bug where when running

    $ while :; do notify-send -a $STR "hi"; done

where the app name, `$STR`, is a string of lengths 2, 5, 7, 13, 18, 20, 22, or 29 (tested up to 30), on commits after [`8eb946d`](https://github.com/Sweets/tiramisu/commit/8eb946dae0e2f98d3850d89e1bb535640e8c3266) (that is, after [`a425e43`](https://github.com/Sweets/tiramisu/commit/a425e4372627242fd9b48bc6d57991c2d867e804) as I couldn't get that one to compile), causes a segfault. This bug was reproduced on two NixOS machines, and on an Arch one as well.

I'm not sure what to do from hereon, but I think this is a good enough reason to block the PR for now. While it's an obscure bug, I still managed to coincidentally trigger it while testing. This should mean that others might be able to coincidentally trigger it as well. I'll open a issue upstream, and we'll see what happens.

File: pkgs/applications/misc/tiramisu/default.nix [hunk 1]
```diff
@@ -24,13 +20,13 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Desktop notifications, the UNIX way";
     longDescription = ''
-    tiramisu is a notification daemon based on dunst that outputs notifications
-    to STDOUT in order to allow the user to process notifications any way they
-    prefer.
+      tiramisu is a notification daemon based on dunst that outputs notifications
```
[the remark this answers is not in the document]
Comment (SuperSandro2000) [reply pkgs/applications/misc/tiramisu/default.nix:23]: Not really. It shouldn't be to long.

**Comment by fortuneteller2k** [comment]
Should be good to merge, since the bug was fixed upstream.

**Pull request merged**